tests/rgw/test_s3.py

```diff
@@ -200,7 +200,7 @@ def rgw_frontend_port(cluster: Ceph, build: str) -> str:
         command = f"{cmd1} | {cmd2}"
     else:
         cmd1 = "grep -e '^rgw frontends' /etc/ceph/ceph.conf"
-        cmd2 = "cut -d ':' -f 2"
+        cmd2 = "cut -d ':' -f 2 | cut -d ' ' -f 1"
```

Comment:
can we add hint of string which is used by `cut` command here.

Reply:
yes... let me do that